format/FlightSql.proto

```diff
@@ -149,6 +149,17 @@ enum SqlInfo {
    */
   FLIGHT_SQL_SERVER_TRANSACTION_TIMEOUT = 101;
 
+  /*
+   * Retrieves a boolean value indicating whether transactions are supported for bulk ingestion. If not, invoking
+   * the method commit in the context of a bulk ingestion is a noop, and the isolation level is
+   * `arrow.flight.protocol.sql.SqlTransactionIsolationLevel.TRANSACTION_NONE`.
+   *
+   * Returns:
+   * - false: if bulk ingestion transactions are unsupported;
+   * - true: if bulk ingestion transactions are supported.
+   */
+   FLIGHT_SQL_SERVER_INGEST_TRANSACTIONS_SUPPORTED = 102;
```

Comment:
Sorry, I should have mentioned: we should also add a value for the ingestion feature itself in general

Reply:
No problem, just added